openstudiocore/ruby/openstudio/sketchup_plugin/OpenStudio.rb

```diff
@@ -20,20 +20,22 @@
 require 'extensions.rb'   # defines the SketchupExtension class
 
 # look for the new plugin
-new_plugin = Sketchup.find_support_file("OpenStudio-config", "Plugins/OpenStudio")
+# This doesn't work anymore. Consider using openstudio -v from the cli
+#new_plugin = Sketchup.find_support_file("OpenStudio-config", "Plugins/OpenStudio")
```

Comment:
kind of. but my main point is that I'm shooting for the installer not to have to do anything beyond dropping the files into the install tree. The openstudio-config should be a thing of the past and instead openstudio ruby bindings are located by relative path based on an assumption of the install tree layout. 

Without this config file though, there is not a handy file to grep for the os version. We may see that version appear in the install tree directory structure when everything gets finalized, but I think the most authentic place to get version is from cli.